src/reference/config/bindings/binding-grpc.md

```diff
@@ -83,10 +83,16 @@ options:
 
 #### options.services
 
+::: warning Deprecated
+This property will be removed in a future release.
+:::
+
 > `array` of `string`
 
 Protobuf service definition filenames, typically with `.proto` filename extension.
 
+Note: this option is deprecated. Use filesystem catalog instead for accessing `.proto` files from the filesystem.
```

Comment:
Recommend we make `filesystem catalog` also link to the reference doc for `catalog-filesystem.md`.